RD-5191 Update Deployments View widget description (#2256)
cloudify-cosmo/docs.getcloudify.org: content/working_with/console/widgets/deploymentsView.md

- the currently selected deployment name
- button to drill-down to deployment details

> - button to go back to the parent deployment
> 
>     That button will only be visible on child environments or services pages.

Reviewer: ```suggestion
- button to go back to parent deployment

    This button is only visible on child environments and services pages.
```

Author: Applied. Changed also some lines below.

- button to drill-down to deployment details
- button to go back to the parent deployment

    This button is only visible on child environments and services pages.

- buttons to drill-down to child environments or services

>     These buttons are not visible on deployments without child environments or services.

Reviewer: ```suggestion
    These buttons are not visible for deployments without child environments or services.
```

Drilling down can happen multiple times if there are more levels in the
deployments hierarchy.

> To go back to the parent page, use "Parent" button in the details pane header

Reviewer 2: Would `details pane header` be clear enough for the user? 🤔 
As the word `details` is used here, it may be indicating a need for clicking` Details` button in order to see the `Parent` button 😁 

WDYT about renaming it to be something like 
```suggestion
To go back to the parent page, use "Parent" button which is placed near the deployment name
```

? 🤔

Author: I used details pane term as there's a section called the same. Added a link to that section to make it more clear. WDYT about that change?

Reviewer 2: Oh, that explains a lot 😁 
Now it looks great :)

Thanks for the explaination and for the adjustment 🎸
